src/Command/ListCollections.php

```diff
@@ -99,11 +100,13 @@ public function __construct(string $databaseName, array $options = [])
     /**
      * Execute the operation.
      *
+     * @return CachingIterator<int, array>
      * @see Executable::execute()
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
     public function execute(Server $server): CachingIterator
     {
+        /** @var Cursor<array> $cursor */
```

Comment:
I believe this was part of Psalm 4 already; the iterator stubs have templates to indicate the key and value types. Since I was working on these classes anyways, I added the annotations for consistency.